jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -122,6 +123,9 @@ public static String getServerInfo()
     private boolean _allowNullPathInContext;
     private Index<ProtectedTargetType> _protectedTargets = Index.empty(false);
     private final List<AliasCheck> _aliasChecks = new CopyOnWriteArrayList<>();
+    private File _tempDirectory;
+    private boolean _tempDirectoryPersisted = false;
+    private boolean _tempDirectoryConfigured = false;
```

Comment:
I don't see a getter for this field?  Also, is this field performing the same function as the context attribute that's being checked at line 88 in `WebInfConfiguration`?